api/applications/serializers.py

```diff
@@ -73,13 +73,15 @@ class ApiOAuth2ApplicationSerializer(ApiOAuthApplicationBaseSerializer):
         required=True,
         validators=[URLValidator()],
         label='Home URL',
+        max_length=200,
```

Comment:
When I cleaned up the applications detail test I found the test `test_partial_update_url_too_long` wasn't working correct, and when I fixed the test it failed, this fixes that case, You'll see below.